geonode/base/models.py

```diff
@@ -1011,6 +1015,17 @@ def keyword_csv(self):
         except BaseException:
             return ''
 
+    def set_bbox_polygon(self, bbox):
+        """
+        Set polygons into bbox_polygon from bbox values
+        """
+        bbox_tup = (bbox[0], bbox[2], bbox[1], bbox[3])
+        bbox_polygon = Polygon.from_bbox(bbox_tup)
+        split_srid = self.srid.split(':')
+        srid_num = int(split_srid[1])
+        bbox_polygon.srid = srid_num
```

Comment:
`bbox_polygon.srid = int(self.srid.split(':')[1])` ?

Reply:
I thought about it, I personally reckon that would be a bit too confusing to understand. but I guess I can add a comment indicating the current string format of srid such as `#to extract srid from 'EPGS: 2700'`?